src/components/Settings.jsx

```diff
@@ -25,17 +25,40 @@ import {
   useUpdateSettingsMutation,
 } from "../store/settingsApi";
 import { useGetProgramsQuery } from "../store/programsApi";
+import {
+  useGetTimetableQuery,
+  useUpdateLayoutVersionMutation,
+} from "../store/timetableApi";
+import config from "../config";
 
 export default function Settings() {
   const userLevel = useSelector((state) => state.auth.userLevel);
+  const { table } = useSelector((state) => state.auth);
+  const { data: timetableData, isSuccess: layoutVersionLoaded } =
+    useGetTimetableQuery(table);
+  const layoutVersion = layoutVersionLoaded
+    ? timetableData.layoutVersion
+    : null;
+
+  if (!layoutVersionLoaded) {
+    return null;
+  }
 
   return (
     <>
       <Container fluid>
         <h2 className="mt-3">Nastavení</h2>
         {userLevel >= level.EDIT && <TimetableTitle />}
-        {userLevel >= level.EDIT && <TimeStep />}
-        {userLevel >= level.EDIT && <Width />}
+        {layoutVersion === "v1" && (
+          <>
+            {userLevel >= level.EDIT && <TimeStep />}
+            {userLevel >= level.EDIT && <Width />}
+          </>
+        )}
+        {userLevel >= level.EDIT &&
+          config.timetableLayoutVersionSwitchingEnabled && (
```

Comment:
why not.. but I have two problems with this approach:
1) this should go ideally into separate PR
2) I need to be able to toggle this in tests, this is the whole motivation behind config slice and override config reducer. So components should not be importing config directly but rather use config slice. Or some other method - I don't care. But I need to be able to write tests for both flag on/off, see here for example: https://github.com/janzarsky/scout-planner/blob/c0a81e029bb4f7c1cc7f09e83ffa09bd0a4fdc5c/cypress/component/EditProgramModal.cy.jsx#L80

so.. maybe leave this for a separate PR and just use the old method for now?

Reply:
I prepared two separate PRs: #730 and #731